components/model_analyzer/tb_dcgm_types/da_exceptions.py

```diff
@@ -1,6 +1,16 @@
 class TorchBenchAnalyzerException(Exception):
     """
-    A custom exception specific to the Triton Model
-    Analyzer
+    A custom exception specific to the TorchBench Model Analyzer
     """
-    pass
\ No newline at end of file
+    pass
+
+
+class TorchBenchAnalyzerException_GPU_Inavailable(Exception):
```

Comment:
Should we rename this to `TorchBenchAnalyzerException_GPU_Unavailable`

Reply:
fixed by https://github.com/pytorch/benchmark/pull/1332/commits/9b74dd72b6c3cea086a6486018044d923cb9bf65